bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/test/java/com/google/cloud/bigtable/hbase/com/google/cloud/bigtable/hbase2_x/adapters/admin/TestTableAdapter2x.java

```diff
@@ -79,9 +82,9 @@ public void testAdaptWithTable(){
     GCRules.GCRule gcRule = GCRULES.maxVersions(maxVersion);
     ColumnFamily columnFamily = ColumnFamily.newBuilder()
             .setGcRule(gcRule.toProto()).build();
-    Table table = Table.newBuilder()
+    Table table = Table.fromProto(com.google.bigtable.admin.v2.Table.newBuilder()
```

Comment:
This seems wrong.  Why not just construct a `mode.Table`?

Reply:
Currently, there is only `model.Table#fromProto` available as a public interface to construct a Table object.

If you want then we can introduce more ways construct `model.Table` object.